src/algebra/order_functions.lean

```diff
@@ -58,6 +58,9 @@ by rw [lt_iff_not_ge]; simp [(≥), max_le_iff, not_and_distrib]
 @[simp] lemma min_lt_iff : min a b < c ↔ a < c ∨ b < c :=
 by rw [lt_iff_not_ge]; simp [(≥), le_min_iff, not_and_distrib]
 
+lemma max_lt_max (h₁ : a < c) (h₂ : b < d) : max a b < max c d :=
+by apply max_lt; repeat { simp [lt_max_iff, h₁, h₂] }
```

Comment:
```suggestion
by apply max_lt; simp [lt_max_iff, h₁, h₂]
```
After the semicolon, the command is applied to all goals.

Reply:
You're right. I changed it from a begin-end block but forgot to remove the repeat. Thanks!